tt_metal/common/bfloat4.hpp

```diff
@@ -58,6 +60,8 @@ inline std::vector<float> unpack_bfp4_tiles_into_float_vec(
     int data_dwords_per_exp_dword_log2 = log2(data_dwords_per_exp * num_exps_in_dword);
     int data_dwords_per_exp_log2 = log2(data_dwords_per_exp);
 
+    uint32_t exp_bit_mask = (tile_HW == 16) ? 0x0 : (tile_HW == 32) ? 0x1 : 0x3;
```

Comment:
Could this be in a common function with an explanation of what 0x0, 0x1, and 0x3 mean?

Also, are there existing define/enums/constants/variables for 0x0, 0x1, 0x3 that could be used instead?

Reply:
I don't think this should be a a seperate function as this won't be used by any other function.

there's no existing enums

Reply:
At least please add a short comment about what these mean then.

Reply:
added, I find it hard to explain the mask here though, it's just get rid of the redundant bits and get a nice pattern to extra bytes from exponent

Reply:
Please have the same comment here as in tt_metal/common/bfloat8.hpp